src/internal/connector/exchange/api/shared.go

```diff
@@ -18,9 +18,14 @@ import (
 // ---------------------------------------------------------------------------
 
 type itemPager interface {
-	getPage(context.Context) (api.DeltaPageLinker, error)
+	// getPage get a page with the specified options from graph
+	getPage(context.Context) (api.PageLinker, error)
+	// setNext is used to pass in the next url got from graph
 	setNext(string)
-	valuesIn(api.DeltaPageLinker) ([]getIDAndAddtler, error)
+	// reset is used to reset delta url in delta pagers
```

Comment:
nit: it's that we're not using the delta at all so maybe a different word would be more meaningful. "Reset" could make it sound like it's going to start at the beginning again
```suggestion
	// reset is used to clear delta url in delta pagers
```